utils/testsuite/testsuite_skiplist.py

```diff
@@ -1900,6 +1900,9 @@
     "cross-realm",
 ]
 
+# Tests for Intl features supported by Hermes
+SUPPORTED_INTL_TESTS = []
```

Comment:
Sorry for the back and forth. @avp noted that the way these lists are set up, we'll end up maintaining a set of individual Intl tests in the main skip list, and then the complement of that set in the `SUPPORTED_INTL_TESTS` list. That adds more stuff to keep track of.

Instead, we can make this list just a list of Intl tests as follows:

```suggestion
# Tests for Intl features
INTL_TESTS = ["test262/test/intl402"]
```

Then whenever Intl is disabled, all Intl tests are disabled, and whenever Intl is enabled, only the Intl tests not in the main skip list are enabled. As we move through the project, we'll only need to update the main skip list.

Reply:
Got it. This definitely makes sense, thanks.